src/insights/position.ts

```diff
@@ -21,4 +22,100 @@ export interface LeaderboardResult {
   meta: PageMeta;
 }
 
-export type QueryGetLeaderboardResponse = Entries<LeaderboardResult>
\ No newline at end of file
+export type QueryGetLeaderboardResponse = Entries<LeaderboardResult>
+
+//Liquidation And ADL
+export interface GetLiquidationAndADLQueryParams extends QueryByPageRequest {
+  sort?: "ASC" | "DESC";
+}
+
+export interface QueryGetLiquidationAndADLResponse {
+  entries: {
+    id:                     string;
+    blockHeight:            string;
+    triggeredBlockHeight:   string;
+    address:                string;
+    market:                 string;
+    side:                   string;
+    price:                  string;
+    quantity:               string;
+    available:              string;
+    filled:                 string;
+    allocatedMarginAmount:  string;
+    allocatedMarginDenom:   string;
+    status:                 string;
+    orderType:              string;
+    initiator:              string;
+    timeInForce:            string;
+    stopPrice:              string;
+    triggerType:            string;
+    isLiquidation:          boolean;
+    isPostOnly:             boolean;
+    isReduceOnly:           boolean;
+    blockCreatedAt:         Date;
+    poolID:                 string;
+    avgFilledPrice:         string;
+    lastUpdatedBlockHeight: string;
+  }[]
```

Comment:
Same issues as `QueryGetLiquidationEngineResponse`

Can consider using the same struct since `QueryGetLiquidationEngineResponse` and `QueryGetLiquidationAndADLResponse` are querying from the same orders table and the results have the same fields